src/scss/components/checkbox/mixins/_hxCheckboxControl.scss

```diff
@@ -0,0 +1,112 @@
+@import "vars";
+@import "hxCheckboxFacade";
+
+//
+// ===== Mixins for <hx-checkbox-control> states =====
+//
+
+// ========== PRISTINE ==========
+@mixin __checkboxControl($psuedo-state: null) {
+  @if $psuedo-state == focus {
+    box-shadow: $focus-glow;
+
+    &::-ms-check {
+      border-color: $cyan-700;
+    }
+
+    ~ label > hx-checkbox {
+      border-color: $cyan-700;
+      box-shadow: $focus-glow;
```

Comment:
These are facade styles. They shouldn't be here.

Reply:
Please move these styles to `hxCheckboxFacade($pseudo-state: focus)` and apply them explicitly in `src/scss/components/checkbox/_index.scss`